pandapower/pypower/dIbr_dV.py

```diff
@@ -60,3 +60,33 @@ def dIbr_dV(branch, Yf, Yt, V):
         It = asarray( Yt * asmatrix(V).T ).flatten()
 
     return dIf_dVa, dIf_dVm, dIt_dVa, dIt_dVm, If, It
+
+def dIbr_dV_new(branch, Yf, Yt, V):
+    # Compute currents.
+    if issparse(Yf):
+        If = Yf * V
+        It = Yt * V
+    else:
+        If = asarray( Yf * asmatrix(V).T ).flatten()
+        It = asarray( Yt * asmatrix(V).T ).flatten()
+
+    vb = range(len(V))
+    diagV = sparse((V, (vb, vb)))
+    diagVnorm = sparse((V / abs(V), (vb, vb)))
+    ib = range(len(If))
+    idxf = abs(If) == 0
+    idxt = abs(It) == 0
+    diagIfnorm = sparse((conj(If) / abs(If), (ib, ib)))
```

Comment:
same here with the try and except block

Reply:
There is a routine in the WLS to avoid using the current magnitude measurements at the first iteration, when currents are initialized as zero. In the other iterations it is quite unlikely to have branch currents exactly equal to 0. If that case happens, we should have some more elaborated routine to deal with it, not just a try and except. It is something that requires some thinking and that would come in one of the next round of updates. For the moment, the priority was to provide a Jacobian for current magnitude measurements that works, at least in most of the cases.